backend/python/app/graphql/mutations/story_mutation.py

```diff
@@ -258,3 +259,23 @@ def mutate(root, info, story_translation_id, user_id):
         except Exception as e:
             error_message = getattr(e, "message", None)
             raise Exception(error_message if error_message else str(e))
+
+
+class ImportStory(graphene.Mutation):
+    class Arguments:
+        story_details = StoryRequestDTO(required=True)
+        story_file = Upload(required=True)
+
+    story = graphene.Field(lambda: StoryResponseDTO)
+
+    def mutate(root, info, story_details, story_file):
+        try:
+            if not services["file"].validate_file(story_file.filename, "docx"):
+                raise Exception("File must be .docx")
```

Comment:
ah shoot - I just checked and the client has some .doc files (application/msword, 97-2003 file format). How difficult would it be to support .doc files in addition to .docx files?